esx_service/cli/vmdkops_admin.xml

```diff
@@ -85,9 +85,68 @@
                 <formatter>table</formatter>
                 <format-parameter name="fields:vdvs">Volume,Datastore,VMGroup,Capacity,Used,Filesystem,Policy,Disk Format,Attached-to,Access,Attach-as,Created By,Created Date</format-parameter>
             </format-parameters>
-            <execute>/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py --output-format=xml volume ls $if{vmgroup, --vmgroup $val{vmgroup}}</execute>
+            <execute>/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py --output-format=xml volume ls $if{vmgroup, --vmgroup '$val{vmgroup}'}</execute>
         </command>
-        <command path="storage.vdvs.volume.set">
+        <command path="storage.guestvol.volume.shortls">
```

Comment:
Yes I thought so.
But on the basis of a flag, -v or (-c as earlier) we cannot decide on which columns to show.
The number of columns to be displayed for a command output is fixed and specified in esxcli xml extension.
The only option here is to introduce a new command.